### ISSUE TITLE
ACE Sitting - No option to sit in interaction menu available and error when manually calling function

### ISSUE THREAD

**Issue by bux** [acemod/ACE3#8576]
**Mods (complete and add to the following information):**
- **CBA** latest
- **ACE3:** `3.14.0`

**Description:**
Option to sit is not available

**Steps to reproduce:**
1. New Mission
2. Place Chairs and player
3. Check addon settings that "ACE Sitting" is enabled
4. Start mission and try to sit down
5. ---
6. Try to call `ace_sitting_fnc_sit`
7. Observe error

**Expected behavior:**
I'm able to sit down on a chair

**Where did the issue occur?**
Editor

**Log Files:** 
https://hastebin.com/xefuqijete.rpt

**Additional context:**

**Screenshots:**
![image](https://user-images.githubusercontent.com/1235520/138393090-5503d788-1f38-41c8-9db2-7ae7319d03f8.png)
![image](https://user-images.githubusercontent.com/1235520/138393127-53987af4-8a7d-4446-9930-4d7569a8fe7f.png)

**Comment by godgutten** [comment]
when will this be fixed?

**Comment by jonpas** [comment]
![image](https://user-images.githubusercontent.com/7935003/142737187-12ec5c39-d7b1-4c40-b810-0680946a0f3e.png)

**Comment by godgutten** [comment]
And what does that mean? Like do we know like when it will be updated or?

**Comment by jonpas** [comment]
No, whenever it is ready. You can follow Release Candidate builds on our Slack (RC1 is available for testing).

**Comment by godgutten** [comment]
Okay. Well, this is kinda a deal-breaker for our unit. But I guess we will wait and see whenever it gets updated.
